shared/router-v2/shim.native.tsx

```diff
@@ -77,6 +77,7 @@ const styles = Styles.styleSheetCreate(
       keyboard: {
         backgroundColor: Styles.globalColors.fastBlank,
         flexGrow: 1,
+        maxHeight: '100%',
```

Comment:
was causing things to get pushed off the screen, including the big divider in inbox